cmd/sebak/cmd/node.go

```diff
@@ -171,6 +163,20 @@ func parseFlagsNode() {
 		flagEndpointString = nodeEndpoint.String()
 	}
 
+	if (len(flagTLSCertFile) < 1) || (len(flagTLSKeyFile) < 1) {
+		fileName := strings.Replace(nodeEndpoint.Host, ":", "_", 1)
+		g := sebaknetwork.NewKeyGenerator(fileName)
+		flagTLSCertFile = g.GetCertPath()
+		flagTLSKeyFile = g.GetKeyPath()
+	}
```

Comment:
I think the choice to auto-generate a certificate should be explicit. It's quite dangerous other, imagine you intend to provide a certificate, but due to some misconfiguration (e.g. wrong shell expansion), you don't.

Reply:
IHMO, It is a good idea to support another cli for generating a certificate. What do you think? :-)